addin-fixed-weights/program/src/instruction.rs

```diff
@@ -21,6 +21,18 @@ pub enum VoterWeightAddinInstruction {
     /// 4. `[signer]` Payer
     /// 5. `[]` System
     SetupVoterWeightRecord { },
+    /// Sets up VoterWeightRecord owned by the program
+    ///
+    /// 0. `[]` Realm account
+    /// 1. `[]` Governing Token mint
+    /// 2. `[]` Governing token owner
+    /// 3. `[writable]` VoterWeightRecord
+    /// 4. `[signer]` Payer
```

Comment:
A `Payer` is not required for this instruction.
Let rename it to `SetVoterPercentage` (like in addin-vesting) and make it equivalent to such in addin-vesting.
One of the questions: can we rewrite serialized enum value for the command?

Reply:
Fixed